### PR TITLE
Update vendor for coredns 1.0.4

### PR DESCRIPTION
**Pull request by corest**

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ import:
 - package: github.com/giantswarm/awstpr
   version: master
 - package: github.com/giantswarm/k8scloudconfig
-  version: master
+  version: v3.0.0-coredns-1.0.4
```
Comment (rossf7) [inline glide.yaml]: Shouldn't this be master?

Comment (rossf7) [reply glide.yaml]: Oof noticed you haven't asked for review! Sorry ignore!

Comment (corest) [reply glide.yaml]: I'm just testing stuff. After testing, I'll change it to the master and update vendor again

File: service/version_bundles.go [hunk 2]
```diff
@@ -204,7 +204,7 @@ func newVersionBundles() []versionbundle.Bundle {
 				},
 				{
 					Name:    "coredns",
-					Version: "1.0.1",
+					Version: "1.0.4",
```
Comment (rossf7) [inline service/version_bundles.go]: Thanks for fixing! <3

File: vendor/github.com/giantswarm/k8scloudconfig/CHANGELOG.md [hunk 3]
```diff
@@ -20,7 +20,6 @@ version directory, and then changes are introduced.
 - Allow setting custom K8s API address for master nodes.
 - Allow setting etcd port.
 - Add node-exporter.
-- Add kube-state-metrics.
```
Comment (rossf7) [inline vendor/github.com/giantswarm/k8scloudconfig/CHANGELOG.md]: Looks like a problem with the metrics being removed.

**Pull request merged**